src/CoreArbiterCommon.h

```diff
@@ -27,8 +28,18 @@
 #define THREAD_BLOCK 1
 #define CORE_REQUEST 2
 
+#define MAX_SUPPORTED_CORES 256
+
 namespace CoreArbiter {
 
+/**
+ * Members of this structure are used by the CoreArbiter to efficiently pass
+ * information to individual threads of a process.
+ */
+struct ThreadCommunicationBlock {
```

Comment:
Are you planning on adding more to this struct? It seems like overkill to wrap a single boolean.

Reply:
@jspeiser , Yes, I think this is a useful construct for communication core-specific information; for example, I may eventually add a flag to indicate whether the core has been forcibly preempted, which can be useful for debugging.

Reply:
Ok, looks good to me! 👍